metadata-db-app/src/cmr/metadata_db/api/provider.clj

```diff
@@ -39,6 +40,15 @@
      :body (json/generate-string providers)
      :headers rh/json-header}))
 
+(defn- validate-consortiums
+  "Throws error if consortiums contain anything other than alphabet, numbers,
+  underscores and spaces. Spaces are used as delimiters."
+  [consortiums]
+  (when (re-find #"[^A-Za-z0-9_ ]" consortiums)
+    (errors/throw-service-error
+      :invalid-data
+      "Invalid consortiums: It contains non-alphanumeric, non-underscore and non-space characters.")))
```

Comment:
Please consider changing the `and` to `or`

Reply:
I think it has to be AND.  when exists (and non-alphanumeric non-space non-underscored)  it's invalid.

Reply:
How about changing the message to: `Invalid consortiums: It can only contain alphanumeric, underscore and space characters.`?

Reply:
Done.